### PR TITLE
Display aws metadata frontend

### PR DESCRIPTION
**Pull request by arbulu89**
Display AWS cloud details. Besides this:
- Rename `Cloud details` to `Provider details`
- Handle the unknown scenario with a `Provider not recognized` message

@abravosuse Putting you as reviewer simply you can validation the images and the visualized data hehe

This is how it looks like:

AWS:
image

Unknown:
image

If the Pill option is not good, we could use a simpler one (or any other suggestion):
image

### CODE REVIEW

File: assets/js/components/HostDetails/ProviderDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import AzureDetails from './AzureDetails';
+import AwsDetails from './AwsDetails';
+import Pill from '@components/Pill';
```
Comment (dottorblaster) [inline assets/js/components/HostDetails/ProviderDetails.jsx:5]: Could you move this import on top under the React one please?

**Review by dottorblaster — APPROVED** [review]
Awesome! One tiny comment and we are done I think :+1:

File: assets/js/components/HostDetails/AwsDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          { title: 'Instance ID', content: provider_data?.instance_id },
+          {
+            title: 'Data disk number',
+            content: provider_data?.data_disk_number,
+          },
+          {
+            title: 'Account ID',
+            content: provider_data?.account_id,
+          },
+          { title: 'Ami ID', content: provider_data?.ami_id },
```
Comment (rtorrero) [inline assets/js/components/HostDetails/AwsDetails.jsx:28]: Very nitpicky: I'd use capital letters for acronyms

File: assets/js/components/HostDetails/AwsDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          {
+            title: 'Region',
+            content: `${provider_data?.region} (${provider_data?.availability_zone})`,
+          },
+          { title: 'Vpc ID', content: provider_data?.vpc_id },
```
Comment (rtorrero) [inline assets/js/components/HostDetails/AwsDetails.jsx:33]: Also capital letters here

**Review by rtorrero — APPROVED** [review]
The pills option for the unknown cloud provider looks good to me.

GJ

**Review by abravosuse — APPROVED** [review]
The pill option looks good to me.

**Pull request merged**